src/main/java/bio/terra/workspace/service/datareference/model/DataReferenceType.java

```diff
@@ -18,8 +22,12 @@
 
   static {
     typeMap.put(DataReferenceType.DATA_REPO_SNAPSHOT, ReferenceTypeEnum.DATA_REPO_SNAPSHOT);
+    typeMap.put(DataReferenceType.GOOGLE_BUCKET, ReferenceTypeEnum.GOOGLE_BUCKET);
+    typeMap.put(DataReferenceType.BIG_QUERY_DATASET, ReferenceTypeEnum.BIG_QUERY_DATASET);
 
     sqlMap.put(DATA_REPO_SNAPSHOT, "DATA_REPO_SNAPSHOT");
```

Comment:
This map is unnecessary. The Enum's built-in `name()` and `validOf` methods will do the same thing. And you can convert to them without changing your interface.

Reply:
The idea of the sql map is to have invariant names, so that even if they refactor the enum names in Java, we can serialize correctly. `.name()` is too smart for this purpose, and would return the new name.

Reply:
Cutting out the `typeMap` seems fine. I'm a little less sure about  `sqlMap` for the reason Jay stated, since this value does get serialized directly.